resources/views/layouts/modal.blade.php

```diff
@@ -1,5 +1,5 @@
 @push('modals-container')
-    <div class="modal fade center-scale"
+    <div class="modal fade center-scale {{$type}}"
```

Comment:
The Bootstrap documentation states that size class selectors should be specified for the `modal-dialog` element. This can be seen at the following link: [Optional Sizes](https://getbootstrap.com/docs/5.3/components/modal/#optional-sizes).

Is there a reason why this was moved?

Reply:
Just wanted to clarify that the `$type` variable is meant for `slide-right`, not for `modal-size`.

I only found the combination `.modal.fade.slide-right` in the CSS, which suggests it's in the wrong spot. It actually works in the modal fade section.

So, I think it makes sense to move it there. Thanks!

Reply:
you can refer to here [`slide-right`](https://github.com/orchidsoftware/platform/blob/b1125e14e88189915eeb8a3532d7c318b82490df/src/Screen/Layouts/Modal.php#L33)

Reply:
Yes, I definitely made a mistake in my reasoning. 
Thank you for pointing that out. Your patch is already in the release.